kdtree/include/pcl/kdtree/kdtree_flann.h

```diff
@@ -64,19 +64,20 @@ namespace pcl
     * \ingroup kdtree 
     */
   template <typename PointT, typename Dist = ::flann::L2_Simple<float> >
-  class KdTreeFLANN : public pcl::KdTree<PointT>
+  class KdTreeFLANN : public pcl::KdTree<PointT, KdTreeFLANN<PointT>>
   {
+    using Self = KdTreeFLANN<PointT, KdTreeFLANN<PointT>>;
```

Comment:
Self is a bad name for parent class

Reply:
Oh, yeah didn't notice I was referencing to base class. WIll update it to `'Base`.